app/views/welcome/index.html.erb

```diff
@@ -111,6 +111,10 @@ body {font-family: "Lato", sans-serif}
         <%= link_to "Be The Change. Learn More", "/learn_more", class: "w3-btn w3-light-green w3-block" %>
       </div>
 
```

Comment:
Same here, change these paths to look like the resources paths, ie `learn_more_path`.